### PR TITLE
Add Toolbars to my portfolio

### PR DESCRIPTION
**Pull request by Ibnballo1**
I added the following to my portfolio website:

- Logo
- Hamburger button
- About Section
- Social media Icons

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+.nav-menu .fas {
+  align-self: center;
+}
+
+.about-me {
+  background-color: var(--sec-bgcolor);
+  border-bottom-left-radius: 70px;
+  height: 98.1vh;
+}
```
Comment (ShahierNashaat) [inline styles.css:53]: - [ ] The about-me section has the proper background image.
- [ ] The content should aligned with flexbox.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    <div class="medium">
+                        <a href="#"><i class="fab fa-medium"></i></a>
+                    </div>
+                    <div class="github">
+                        <a href="#"><i class="fa fa-github"></i></a>
+                    </div>
+                    <div class="angellist">
+                        <a href="#"><i class="fa fa-angellist"></i></a>
+                    </div>
+                </div>
```
Comment (ShahierNashaat) [inline index.html:67]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Ibnballo1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please remove the eslint from your project.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibnballo1** [comment]
@ShahierNashaat , I sent feedback together with my code request review that you should ignore the eslint that I  will remove that after I merged the feature branch with the main branch. So I don't know why you requested I remove it, perhaps you did not receive the feedback...
Also, I used flex box in my CSS, so what am I doing wrong.

as for the third, I will change it to a list item

**Comment by ShahierNashaat** [comment]
Hey @Ibnballo1  
- Kindly be aware that in this project it is required that you use only HTML and CSS Linters and they don't have eslint also your pull request should pass all the linters checks and as you got the eslint linter, not all the checks passed.
- Also Kindly be aware that the about-me section should have a display: flex; as the header.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Ibnballo1  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Design looks similar to that of the figma template 🥳 
- Linter checks are passing 👍 
- Implemented suggested changes by the previous reviewer

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- Your README is looking awesome 🍾  however, it is referencing Hello Microverse instead of your mobile portfolio as seen in this image 👀 
image
The link in your README for the MIT license is displaying raw and is not working as seen in the image below. Please fix that 😃 
image

**Pull request merged**